Battery.elm

```diff
@@ -1,35 +1,48 @@
 module Battery exposing (Battery, recommend)
 
-import Usage exposing (Usage)
+import List exposing (head, map, repeat, sortBy, sum)
 import Load exposing (getAmps)
-import List exposing (sum, map, repeat, sortBy, head)
+import Usage exposing (Usage)
+
+
 -- MODEL
 
+
 type alias Battery =
     { name : String
     , maxDischargePercentage : Int
     , ah : Int
     , price : Float
     }
 
-liFePO4 = Battery "LiFePO4" 80
 
 batteries : List Battery
 batteries =
+    let
+        liFePO4 =
+            Battery "LiFePO4" 80
+    in
     [ liFePO4 40 215.04
     , liFePO4 60 309.12
     , liFePO4 90 483.84
     ]
 
-recommend : Int -> List Usage -> Maybe ( List Battery )
+
+recommend : Int -> List Usage -> Maybe (List Battery)
 recommend volt load =
     let
-      getSmallestPack : Float -> Battery -> List Battery
-      getSmallestPack amps battery =
-          repeat ( ceiling ( amps / toFloat battery.ah ) ) battery
+        getSmallestPack : Float -> Battery -> List Battery
+        getSmallestPack amps ({ ah } as battery) =
+            (amps / toFloat ah)
+                |> ceiling
+                |> flip repeat battery
     in
-        head ( sortBy packPrice ( map ( getSmallestPack ( getAmps volt load ) ) batteries ) )
+    batteries
+        |> map (getSmallestPack <| getAmps volt load)
+        |> sortBy packPrice
+        |> head
```

Comment:
in my experience, this is the most readable style for beginners.

Reply:
This looks way better, thanks!